lib/game.rb

```diff
@@ -53,16 +52,7 @@ def play_again?
     # asks if the player wants to restart the game
     @display.output(PLAY_AGAIN)
     answer = @display.input
-    if answer === "yes" || answer === "y"
-      restart
-    end
```

Comment:
Seems like you're getting input at various points from the user.  You could have an object solely responsible for getting input from the user, that could also be initialized with a validator.  You could pass this object to other objects (ex. game, player, etc.) that need input from the user.

Reply:
Isn't that the display?